app/Whisky.php

```diff
@@ -50,9 +49,7 @@ public static function base_path(string $path = ''): string
 
     public static function readConfig(string $key): string|array|null
     {
-        $path = self::cwd('whisky.json');
-
-        $cfg = json_decode(File::get($path), true);
+        $cfg = FileJson::make(self::cwd('whisky.json'))->read();
```

Comment:
I'd like to refactor how this is called eventually to make it a bit cleaner, but for now it gets us validated so I'm going to just leave it.